### DISCUSSION TITLE
Potential Audience goer Searching for Non-traditional Performance in Toronto

### DISCUSSION THREAD

**Discussion by mariel-marshall** [pair-cg/general-contributions#77] [User Stories]
# 1. Motivating Story
_What's the story here? Please describe why you have a particular need.

I am an audience goer that enjoys attending unconventional performance experiences happening in real life (not online) in Toronto and I want to find out what experiences are upcoming so that I can get tickets and attend. The experience must take place In Real Life, and be something I can go to in-person. I'm not interested in online events right now.  (By "unconventional", I mean works that are being staged in nontraditional ways, including immersive, experiential, experimental or site-specific works.) In order to adjust my expectations about the event in advance, I generally like to get a good feel for the size of the team producing it, and what backing/funding exists to support the project. E.g. is this an experimental sound installation happening in someone's backyard on zero budget, or a production as part of Luminato happening site specifically at the Hern Generating Station. I like to take photographs so I prioritize things that allow for photography (before/during/after the event) and which have interesting photographic subject matter.

# 2. Need:
_Now the why is out of the way, what is it you actually need?_

To be aware of upcoming immersive, experiential, experimental and site-specific performance experiences (of any art discipline) happening in real life in Toronto, within an hour driving radius of College and Bathurst Street that would be suitable for me to attend with friends and family (all adults). These experiences must be Covid-safe (ie. they follow health and safety measures such as physical distancing and mask usage) happening weekday evenings or weekends. (By upcoming performance I mean a performance scheduled within a month from now.)

**Time Frame**
- the date of the experience is happening within a month from now
- taking place between 6pm - 10pm EDT weekdays, or 10am - 11pm EDT Sat/Sundays

**Form**
 immersive OR  experimental OR site-specific

**Distance from me:** 
- 40km

**Cost**
- 0-$45 

**Permission to photograph**
- yes

**IRL**
- experience takes place In Real Life, in-person. 

**Participant Interactivity Level**
"none OR low OR medium OR high"
My definitions for interactivity levels
1. None - Sit back and experience
2. Low - Respond to performance eg. dancing at a concert, or
3. Medium - Influence the performers eg. giving suggestions
4. High - Become a performer or takeover the performance eg. reading a role, singing on stage

**Production Format**
"Sandbox OR Tracked OR One-on-One"
My definitions for production formats:
- Sandbox / Open World (Like a free-roaming video game)
- Tracked (Linear flow, like a theme park dark ride – but with or without vehicles)
- One-on-One (Intimate experiences designed for the smallest of audiences)

**Participant agency**
"Narrative OR Investigative OR Emotional OR Traversal OR Co-Creation"
My definitions for participant agency:
- Narrative (Power to change story)
- Investigative (Power to reveal, but not change story)
- Emotional (Power to influence performance, but not change story)
- Traversal (Freedom to move through space)
- Co-Creation (Active Improvisation)

**Age Appropriateness**
target audience age >= 18  (i.e., I'm open to all experiences, except those designed for children)

**Age Restrictions**
None

**Food & Beverage available**
- yes

**Alcohol Restrictions**
- none

**Content warnings** 
None    (i.e., I'm okay with mature content)

**Sensory warnings**
None (ie. I'm okay with use of fog or smoke, strobe lights)

# 3. Action:
_What is an action that can be taken towards fulfilling that need?_

- search online for experiences that meet my criteria
- read arts and entertainment blogs / magazines I know about to try and discover when something is programmed that meets my criteria
- research (online) which companies in the city are most likely to program this kind of work
- subscribe to these company's newsletters to get notified of their programming  (e.g. Outside the March, Volcano, etc.)
- subscribe to media outlets that cover non-traditional works (e.g., NoPro)
- follow immersive theatre influencers on social media

# 4. Desired Outcome:
_What is the desired outcome?_

To have an amazing experience at an arts & entertainment event with my friends/family.

# 5. Perspective:
_From what perspective is this user story coming from?_

Audience member

# 6. Stakeholder Group:
_Which stakeholder groups are impacted by this User Story?_

Audience member
Artists & Producers & Presenters
Theatre influencers
Entertainment bloggers
Arts & Culture journalists

# 6. Additional info:
_Here's more information._

**Comment by drosu** [comment]
Hi @mariel-marshall 
Can you describe the suitability parameters for your context.
Thanks!

**Reply by mariel-marshall** [reply]
I added that up under "needs"
